deliver/lib/deliver/upload_metadata.rb

```diff
@@ -175,17 +175,26 @@ def upload(options)
 
                        non_localized_version_attributes['earliestReleaseDate'] = date
                        Spaceship::ConnectAPI::AppStoreVersion::ReleaseType::SCHEDULED
-                     elsif options[:automatic_release]
+                     elsif options[:automatic_release] == true
                        Spaceship::ConnectAPI::AppStoreVersion::ReleaseType::AFTER_APPROVAL
-                     else
+                     elsif options[:automatic_release] == false
                        Spaceship::ConnectAPI::AppStoreVersion::ReleaseType::MANUAL
                      end
-      non_localized_version_attributes['releaseType'] = release_type
+      if release_type.nil?
+        UI.important("Release type will not get because Neither `automatic_release` nor `auto_release_date` are provided. Please explicitly set one of these options if you need a release type set")
```

Comment:
? 🤔 😅 
```suggestion
        UI.important("Release type will not be set because neither `automatic_release` nor `auto_release_date` were provided. Please explicitly set one of these options if you need a release type set")
```

Reply:
I’M SO BAD AT NON-PROGRAMMING WORDS